pkg/test/integration/common/framework.go

```diff
@@ -1197,6 +1197,22 @@ func (c *IntegrationTestFramework) Cleanup() {
 				Update(ctx, mcmDeploymentOrigObj, metav1.UpdateOptions{})
 			return updateErr
 		})
+
+		// scale back up the MCM deployment to 1 in the Control Cluster
+		mcmDeployment, err := c.ControlCluster.Clientset.
+			AppsV1().
+			Deployments(mcmDeploymentOrigObj.Namespace).
+			Get(ctx, mcmDeploymentOrigObj.Name, metav1.GetOptions{})
+		if err != nil {
+			log.Println(err.Error())
+		}
+		if *mcmDeployment.Spec.Replicas == 0 {
```

Comment:
is dereferencing needed? I am not able to see the type of returned value, so unsure

Reply:
Yes, @himanshu-kun. `Replica` field is of type `int32`.

Reply:
ok so it means dereferencing is not req

Reply:
Sorry, that was a typo. It is of type `*int32`. So, deferencing is needed.